spec/features/favorites/user_can_view_favorites_index_page_spec.rb

```diff
@@ -66,7 +66,24 @@
        click_link("#{@pet2.name}")
        expect(current_path).to eq("/pets/#{@pet2.id}")
      end
+  end
+
+  scenario "see a message regarding no favorited pets if none have been favorited" do
+    visit '/favorites'
+
+    expect(page).to_not have_content("You have not favorited any pets.")
+    expect(page).to have_content("Patra")
+
+    visit "/pets/#{@pet1.id}"
+    click_button("Remove this pet from favorites.")
+
+    visit "/pets/#{@pet2.id}"
+    click_button("Remove this pet from favorites.")
+
+    visit '/favorites'
 
+    expect(page).to have_content("You have not favorited any pets.")
+    expect(page).to_not have_content("Patra")
```

Comment:
Test to confirm messaging if no pets have been favorited. Just went ahead and added this to the already existing user can view favorites page. If we want to split it up, that's totally cool as well.

Reply:
No need to split it up; this addition makes the testing more robust.